client/my-sites/site-settings/form-general.jsx

```diff
@@ -486,8 +437,6 @@ class SiteSettingsFormGeneral extends Component {
 					</form>
 				</Card>
 
-				{ this.dateTimeFormat() }
```

Comment:
Some further cleanup of `form-general` would be recommended here:

* Removing unnecessary fields from `defaultSettings` and `formSettings`, for example `date_format`, `time_format` and `start_of_week`.
*  Removing the GMT offset logic from the `wrapSettingsForm` call.